packages/aws-cdk-lib/package.json

```diff
@@ -453,6 +453,8 @@
     "./aws-cognito": "./aws-cognito/index.js",
     "./aws-config": "./aws-config/index.js",
     "./aws-connect": "./aws-connect/index.js",
+    "./aws-connectcampaigns": "./aws-connectcampaigns/index.js",
```

Comment:
You removed something from code, no additions, no edits. These two changes do not belong in the same PR.